src/components/sideMenu/view/sideMenuView.js

```diff
@@ -20,53 +20,46 @@ import styles from './sideMenuStyles';
 // Images
 const SIDE_MENU_BACKGROUND = require('../../../assets/side_menu_background.png');
 
-class SideMenuView extends Component {
-  /* Props
-   * ------------------------------------------------
-   *   @prop { type }    name                - Description....
-   */
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      menuItems: props.isLoggedIn ? MENU.AUTH_MENU_ITEMS : MENU.NO_AUTH_MENU_ITEMS,
-      isAddAccountIconActive: false,
-      storageT: 'R',
-    };
-  }
+const SideMenuView = ({
+  currentAccount,
+  isLoggedIn,
+  intl,
+  handleLogout,
+  accounts,
+  switchAccount,
+  navigateToRoute,
+}) => {
+  const [menuItems, setMenuItems] = useState(
+    isLoggedIn ? MENU.AUTH_MENU_ITEMS : MENU.NO_AUTH_MENU_ITEMS,
+  );
+  const [isAddAccountIconActive, setIsAddAccountIconActive] = useState(false);
+  const [storageT, setStorageT] = useState('R');
 
   // Component Life Cycles
-  componentDidMount() {
+  useEffect(() => {
+    let _isMounted = false;
     getStorageType().then(item => {
-      this.setState({ storageT: item });
+      if (!_isMounted) {
```

Comment:
i guess this if is not necessary, it will always true. maybe you wanted to check item

Reply:
It is necessary in some cases, function might be async and component might be unmounted already. But, I will fix it, it should have unsubscribe `return`.